documentation-website/Writerside/topics/DSL-CRUD-operations.topic

```diff
@@ -498,35 +401,36 @@
             This can be accomplished by using one of the following functions:
         </p>
         <list>
-            <li><code>insertReturning()</code></li>
-            <li><code>upsertReturning()</code></li>
-            <li><code>updateReturning()</code></li>
-            <li><code>deleteReturning()</code></li>
+            <li>
+                <a href="https://jetbrains.github.io/Exposed/api/exposed-core/org.jetbrains.exposed.sql/insert-returning.html">
+                    <code>insertReturning()</code>
+                </a>
+            </li>
+            <li>
+                <a href="https://jetbrains.github.io/Exposed/api/exposed-core/org.jetbrains.exposed.sql/upsert-returning.html">
+                    <code>upsertReturning()</code>
+                </a>
+            </li>
+            <li>
+                <a href="https://jetbrains.github.io/Exposed/api/exposed-core/org.jetbrains.exposed.sql/update-returning.html">
+                    <code>updateReturning()</code>
+                </a>
+            </li>
+            <li>
+                <a href="https://jetbrains.github.io/Exposed/api/exposed-core/org.jetbrains.exposed.sql/delete-returning.html">
+                    <code>deleteReturning()</code>
+                </a>
+            </li>
         </list>
         <p>
             Each of them take a list of the required table columns
             as an argument. If not provided, all table columns will be returned by default:
         </p>
 
-        <code-block lang="kotlin">
-            object Projects : Table(&quot;projects&quot;) {
-                val title = varchar(&quot;title&quot;, 64)
-                val budget = integer(&quot;budget&quot;)
-                val created = datetime(&quot;created&quot;).defaultExpression(CurrentDateTime)
-            }
-
-            // returns all table columns by default
-            val created: LocalDateTime = Projects.insertReturning {
-                it[title] = &quot;Project A&quot;
-                it[budget] = 100
-            }.single()[Projects.created]
-
-            val updatedBudgets: List&lt;Int&gt; = Projects.updateReturning(listOf(Projects.budget)) {
-                it[budget] = Projects.budget.times(5)
-            }.map {
-                it[Projects.budget]
-            }
-        </code-block>
+        <code-block lang="kotlin"
+                    src="exposed-dsl/src/main/kotlin/org/example/examples/ModifiedRowsExamples.kt"
+                    include-symbol="created, updatedBudgets"
+        />
```

Comment:
Based on what I'm seeing in the preview, there may be a symbol naming ambiguity issue here. `ModifiedRowsExamples.kt` has 2 `created`, 1 in the table object and one in `returnData()`. And it's the column `created` that is showing in the code block, which I guess is not what we want to show?